fluentd/configs.d/filter-post-z-retag-two.conf

```diff
@@ -18,4 +22,5 @@
   rewriterule1 message .+ output_tag
   rewriterule2 MESSAGE .+ output_tag
   rewriterule3 log .+ output_tag
+  rewriterule4 message !.+ output_tag
```

Comment:
The same question: 'We want to retag all the logs tagged as "journal.** system.var.log** default openshift openshift-infra audit.log**". Could it be done with one rule instead of 4?'